spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -1,21 +1,37 @@
 module Spaceship::TestFlight
   class Client < Spaceship::Client
+    ##
+    # Spaceship HTTP client for the testflight API.
+    #
+    # This client is solely responsible for the making HTTP requests and
+    # parsing their responses. Parameters should be either named parameters, or
+    # for large request data bodies, pass in anything that can resond to
+    # `to_json`.
+    #
+    # Each request method should validate the required parameters. A required parameter is one that would result in 400-range response if it is not supplied.
+    # Each request method should make only one request. For more high-level logic, put code in the data models.
+
     def self.hostname
       'https://itunesconnect.apple.com/testflight/v2/'
     end
 
+    ##
+    # @!group Build trains API
+    ##
+
     # Returns an array of all available build trains (not the builds they include)
     def get_build_trains(app_id: nil, platform: nil)
       assert_required_params(__method__, binding)
+
       platform ||= "ios"
       response = request(:get, "providers/#{team_id}/apps/#{app_id}/platforms/#{platform}/trains")
       handle_response(response)
     end
 
     def get_builds_for_train(app_id: nil, platform: nil, train_version: nil)
       assert_required_params(__method__, binding)
-      platform ||= "ios"
 
+      platform ||= "ios"
```

Comment:
`platform` will never be nil because of the assertion above. Perhaps we should set `platform: 'ios'` in the parameter names, or get rid of this.